sample/HelloWorld/public/js/helloworld.js

```diff
@@ -1,29 +1,22 @@
-// Initialize an OpenTok Session object
+// Initialize an OpenTok Session object.
 var session = OT.initSession(sessionId);
 
-// Initialize a Publisher, and place it into the element with id="publisher"
+// Initialize a Publisher, and place it into the 'publisher' DOM element.
 var publisher = OT.initPublisher(apiKey, 'publisher');
 
-// Attach an event handler for when the session dispatches the 'streamCreated' event.
 session.on('streamCreated', function(event) {
-    // This function runs when another client publishes a stream (eg. session.publish())
-
-    // Subscribe to the stream that caused this event, put it inside the DOM element with id='subscribers'
-    session.subscribe(event.stream, 'subscribers', {
-        insertMode: 'append'
-    });
+  // Called when another client publishes a stream.
+  // Subscribe to the stream that caused this event.
+  session.subscribe(event.stream, 'subscribers', { insertMode: 'append' });
 });
 
-// Connect to the Session using the 'apiKey' of the application and a 'token' for permission
+// Connect to the session using your OpenTok API key and the client's token for the session
 session.connect(apiKey, token, function(error) {
-    // This function runs when session.connect() asynchronously completes
```

Comment:
didn't think this was helpful? some JS newbs don't completely grasp the concept of a callback or completion handler.